infrastructure/cdn-in-a-box/traffic_vault/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN rm -rfv /etc/riak/prestart.d/* /etc/riak/poststart.d/*
 RUN echo 'APT::Install-Recommends 0;' >> /etc/apt/apt.conf.d/01norecommends \
  && echo 'APT::Install-Suggests 0;' >> /etc/apt/apt.conf.d/01norecommends \
  && apt-get update \
- && DEBIAN_FRONTEND=noninteractive apt-get install -y net-tools ca-certificates dnsutils gettext-base \
+ && DEBIAN_FRONTEND=noninteractive apt-get --no-install-recommends install -y net-tools ca-certificates dnsutils gettext-base \
```

Comment:
As with the `dns` Dockerfile,

```dockerfile
RUN echo 'APT::Install-Recommends 0;' >> /etc/apt/apt.conf.d/01norecommends \
 && echo 'APT::Install-Suggests 0;' >> /etc/apt/apt.conf.d/01norecommends \
```
already updates the APT config to skip package recommendations, so there is no need to pass the `--no-install-recommends` option with every apt or apt-get command.